data-serving/data-service/schemas/cases.indexes.json

```diff
@@ -31,12 +31,12 @@
         }
     },
     {
-        "name": "uploadIdIdx",
+        "name": "uploadIdsIdx",
         "key": {
-            "caseReference.uploadId": -1
+            "caseReference.uploadIds": -1
         },
         "partialFilterExpression": {
-            "caseReference.uploadId": {
+            "caseReference.uploadIds": {
                 "$exists": true
```

Comment:
Since this is an array, do we now need to check that the length of the array is >0, rather than it exists?

Reply:
Looks like this can be done by checking in the first element in the array exists, done.